guides/source/working_with_javascript_in_rails.md

```diff
@@ -148,6 +148,7 @@ and Rails has got your back in those cases.
 Because of Unobtrusive JavaScript, the Rails "Ajax helpers" are actually in two
 parts: the JavaScript half and the Ruby half.
 
+Unless you have disabled the [Asset Pipeline](asset_pipeline.html),
```

Comment:
putting two links next to each other feels a bit awkward, more of the focus is on rails.js in this sentence.